installer/docker/Dockerfile

```diff
@@ -84,17 +91,17 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.schema-version=$VCS_TAG
 
 # install hnn-core
-RUN sudo pip install hnn-core
+RUN pip install hnn-core
+
+RUN sudo apt-get update && \
+    sudo apt-get install -y neuron-dev
```

Comment:
Do you still need sudo here? Looks like you've removed it elsewhere.

Reply:
For some of the steps, I would encounter a `permissions` error if the sudo is removed, so I've kept it in some of the commands